Makefile

```diff
@@ -1,9 +1,26 @@
 export TF_ACC=1
 
+# Values to install the provider locally for testing purposes
+HOSTNAME=registry.terraform.io
+NAMESPACE=mackerelio-labs
+NAME=mackerel
+BINARY=terraform-provider-${NAME}
+VERSION=9.9.9
+OS_ARCH=darwin_amd64 # change according to your environment
```

Comment:
Could `go env` replace here? or else able to use it just in a recipe directly?

```makefile
OS_ARCH=$(shell go env GOHOSTOS)_$(shell go env GOHOSTARCH)

all:
	echo $(OS_ARCH)
```

In my Linux box:

```console
$ make
linux_amd64
```